src/utils/compareObjectsKeysLength/index.ts

```diff
@@ -0,0 +1,9 @@
+const compareObjectsKeysLength = (object1: {}, object2: {}) => {
```

Comment:
[Nit] Prefer `Record<string, unknown>`, `{}` will type it as a literal empty object.

<img width="194" alt="Screen Shot 2020-11-09 at 12 36 35 PM" src="https://user-images.githubusercontent.com/13544620/98582309-6cbaa500-2288-11eb-9003-f0108a370bf9.png">